onnx/defs/operator_sets.h

```diff
@@ -951,6 +951,8 @@ class OpSet_Onnx_ver14 {
 };
 
 inline void RegisterOnnxOperatorSetSchema() {
+  // 0 means all versions of ONNX schema have been loaded
+  OpSchemaRegistry::Instance()->SetLoadedSchemaVersion(0);
```

Comment:
why not set the default valur to 0 so that you dont need to do anything here

Reply:
Actually I set the default value as -1 to represent that the schema has not been loaded yet. If so, setting 0 here is required.

Regarding to a scenario in any Runtime: if Runtime wants to load specified schema during inference, it needs to know whether opset schema has been loaded or not to prevent conflict. This can be done in either ONNX or Runtime. I choose to implement here because I think this thing should be under ONNX-level and it can be reused by every Runtime frameworks.

Reply:
I dont understand... what is the purpose of setting the schema version to 0 ?

Reply:
Default is -1. Setting 0 here to let OpSchemaRegistry know that ONNX opset schema has been **fully** loaded. It would be useful for Runtime to detect whether OpSchemaRegistry has loaded ONNX schema or not to prevent loading conflict.